inc/namespace.php

```diff
@@ -222,3 +226,17 @@ function hide_welcome_panel( $value, int $user_id, string $meta_key ) {
 
 	return [ 0 ];
 }
+
+/**
+ * Prevent direct web access to wp-admin/install.php.
+ */
+function prevent_web_install() {
+	if ( $_SERVER['REQUEST_URI'] !== '/wp-admin/install.php' ) {
```

Comment:
I guess we probably can't detect this a different way? Like, some sort of `WP_INSTALLING` constant?

Reply:
Not that I can see